src/main/io/displayport_hott.c

```diff
@@ -141,6 +141,11 @@ displayPort_t *displayPortHottInit()
     return &hottDisplayPort;
 }
 
+void hottDisplayportRegister()
+{
+    cmsDisplayPortRegister(displayPortHottInit());
```

Comment:
Not sure this needs to be exported as part of `displayport_hott` .. just adding `cmsDisplayPortRegister(displayPortHottInit());` to the telemetry initialization should be sufficient, as `displayPortHotInit` is already exported.

Reply:
I don't want a CMS dependency in hott.c, because the "display" shouldn't know CMS and CMS  shouldn't know its displays directly. Both should only know displayport. 
Displayport is the interface to CMS, so all input/output goes through it.

Reply:
I guess it's just a matter of philosophy.  To me, it seems that before the registration burden was carried by `fc` and it is now being transferred to the `telemetry` system.  The init function is implemented to return a pointer so the implementing system can obtain a reference to the displayport and register it with the producer (cms).  If the displayport is to be tightly coupled with CMS and telemetry, there is no reason to export the init function and it should be declared statically.

Reply:
Just my opinion so take it as you'd like :)  I'm not here to judge or debate your philosophy.  Just sharing my thoughts.